src/guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -18,13 +18,13 @@ You can upgrade your Magento application from the command line if you installed
 {:.bs-callout-info}
 Do not use this method to upgrade if you cloned the Magento 2 GitHub repository. Instead, see [Update the Magento application][] for upgrade instructions.
 
-The following instructions show you how to upgrade Magento using Composer. Magento 2.4.2 introduced support for Composer 2. If you are attempting to upgrade from Magento <2.4.1, you must first upgrade to a version of Magento that is compatible with Composer 2 (for example, 2.4.2) using Composer 1 _before_ upgrading to Composer 2 for Magento >2.4.2 upgrades.
+Magento 2.3.7 introduced support for Composer 2. Composer 1.x will work but it is no longer in development and we recommend upgrading to version 2.
```

Comment:
Have you checked with engineering about removing this messaging? I only ask because I think it's important for anyone upgrading their version of Magento; not necessarily installing a fresh instance. In other words, you must upgrade to a version of Magento compatible with Composer 2 **using Composer 1** befoore upgrading your version of Composer. That seems like an important, albeit obscure, detail that people coould get stuck oonn if we don't document it.

Cc: @Rrego6

Reply:
Confirmed with Max L. and Misha. 2.3.7 will work with both and doesn't have the strict upgrade order that 2.4.2 has.